definitions/consented/json/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -4555,5 +4555,19 @@
     "CaseFieldID": "childrenInfo",
     "UserRole": "caseworker-divorce-systemupdate",
     "CRUD": "CRUD"
+  },
+  {
+    "LiveFrom": "01/06/2020",
+    "CaseTypeID": "FinancialRemedyMVP2",
+    "CaseFieldID": "paymentHistory",
+    "UserRole": "caseworker-divorce-financialremedy-courtadmin",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/06/2020",
+    "CaseTypeID": "FinancialRemedyMVP2",
+    "CaseFieldID": "paymentHistory",
+    "UserRole": "caseworker-divorce-systemupdate",
```

Comment:
I presume this is being provided permissions for integration with Bulk Scanning?

Reply:
quoting https://tools.hmcts.net/confluence/display/BSP/CCD+config+roles+setup+and+rules: `Application CCD configs don't need bulkscan roles, they need to maintain caseworker roles only.`. To me that means we just give permissions to caseworkers - bulkscan role is required in bulkscan configs to trigger bulkscan events.